### DISCUSSION TITLE
How to check spelling and grammar ?

### DISCUSSION THREAD

**Discussion by lipatovroman** [Sequel-Ace/Sequel-Ace#608] [Q&A]
Hello!
Is it possible to check spelling and grammar for all cells in column?

![2020-12-09 21-09-32](https://user-images.githubusercontent.com/6747127/101675801-ef3eac00-3a62-11eb-8506-30e8ce560f12.jpg)

**Comment by Kaspik** [comment]
I don't think we currently have anything like that. The grammar and spelling is provided by system over `String` - column is array of values...

**Reply by lipatovroman** [reply]
Ok, let it be.
But system over String option not working.
Explain.
When I edit tables in numbers I can see  grammar and spelling checking.
![2020-12-16 10-55-27](https://user-images.githubusercontent.com/6747127/102326577-3d850b00-3f8d-11eb-9c95-d869428f52ec.jpg)

But it doesn't work in Sequel Ace.

**Comment by Kaspik** [comment]
If you double click the field, it will change to "editing" mode, where you are accessing the string itself. Right click on the string and contextual menu should give you spelling and grammar option?

**Reply by lipatovroman** [reply]
Yes, I need:
1. Click on value to change cell in "editing" mode.
2. Right click and find in contextual menu spelling and grammar option.
3. Right click again and find in contextual menu option Check Spelling.
But Idea is to check all cells in row.  

Watch this video - https://biblsoft.ru/images/monopics/2020-12-16_11-10-39.mp4

**Reply by Kaspik** [reply]
Ahh, yeah, that's unfortunately not possible right now, sorry.
